docs/source/usage.rst

```diff
@@ -0,0 +1,364 @@
+	   },
+	   {
+		'data': {'bicycle': {'serial_number': 'fgh', manufacturer: 'Bicycle Inc.'}},
+		'id': '748f6c30daaf771c9020d84db9ad8ac4d1f7c8de7013db55e16f10ba090f7013'
+	   }
+	]
+
+
+This call returns all the assets that match the string 'Bicycle Inc.', sorted by text score, as well as the asset id. This is the same id of the transaction that created the asset.
+
+
+
+Recap: Asset Creation & Transfer
+--------------------------------
+
+.. code-block:: js
+
+	const driver = require('bigchaindb-driver')
+
+	// BigchainDB server instance or IPDB (e.g. https://test.ipdb.io/api/v1/)
+	const API_PATH = 'http://localhost:9984/api/v1/'
+
+	// Create a new keypair for Alice and Bob
+	const alice = new driver.Ed25519Keypair()
+	const bob = new driver.Ed25519Keypair()
+
+	console.log('Alice: ', alice.publicKey)
+	console.log('Bob: ', bob.publicKey)
+
+	// Define the asset to store, in this example
+	// we store a bicycle with its serial number and manufacturer
+	assetdata = {
+		'bicycle': {
+			'serial_number': 'cde',
+			'manufacturer': 'Bicycle Inc.',
+		}
+	}
+
+	// Metadata contains information about the transaction itself
+	// (can be `null` if not needed)
+	// E.g. the bicycle is fabricated on earth
+	metadata = {'planet': 'earth'}
+
+	// Construct a transaction payload
+	const txCreateAliceSimple = driver.Transaction.makeCreateTransaction(
+		assetdata,
+		metadata,
+
+		// A transaction needs an output
+		[ driver.Transaction.makeOutput(
+				driver.Transaction.makeEd25519Condition(alice.publicKey))
+		],
+		alice.publicKey
+	)
+
+	// Sign the transaction with private keys of Alice to fulfill it
+	const txCreateAliceSimpleSigned = driver.Transaction.signTransaction(txCreateAliceSimple, alice.privateKey)
+
+	// Send the transaction off to BigchainDB
+	const conn = new driver.Connection(API_PATH)
+
+	conn.postTransaction(txCreateAliceSimpleSigned)
+		// Check status of transaction every 0.5 seconds until fulfilled
+		.then(() => conn.pollStatusAndFetchTransaction(txCreateAliceSimpleSigned.id))
+		.then(retrievedTx => console.log('Transaction', retrievedTx.id, 'successfully posted.'))
+		// Check status after transaction has completed (result: { 'status': 'valid' })
+		// If you check the status of a transaction to fast without polling,
+		// It returns that the transaction is waiting in the 'backlog'
+		.then(() => conn.getStatus(txCreateAliceSimpleSigned.id))
+		.then(status => console.log('Retrieved status method 2: ', status))
+
+		// Transfer bicycle to Bob
+		.then(() => {
+			const txTransferBob = driver.Transaction.makeTransferTransaction(
+				// signedTx to transfer
+				txCreateAliceSimpleSigned,
+				// metadata
+				{price: '100 euro'},
+				[driver.Transaction.makeOutput(driver.Transaction.makeEd25519Condition(bob.publicKey))], 
+				0)
+			
+			// Sign with alice's private key
+			txTransferBobSigned = driver.Transaction.signTransaction(txTransferBob, alice.privateKey)
```

Comment:
Needs `let`?

Reply:
This still isn't defined anywhere higher up, so needs a variable declaration? `let, const, var`?